packages/client/wallets/smart-wallet/src/api/CrossmintWalletService.ts

```diff
@@ -22,36 +22,28 @@ export class CrossmintWalletService extends BaseCrossmintService {
         return this.fetchCrossmintAPI(
             `sdk/smart-wallet/versions?chain=${chain}`,
             { method: "GET" },
-            `Error getting smart wallet version configuration. Please contact support`,
+            "Error getting smart wallet version configuration. Please contact support",
             user.jwt
         );
     }
 
     async getPasskeySigner(user: UserParams): Promise<PasskeySignerData | null> {
-        try {
-            const signers = await this.fetchCrossmintAPI(
-                `sdk/smart-wallet/signers?type=passkeys`,
-                { method: "GET" },
-                "Error fetching passkey validator signer. Please contact support",
-                user.jwt
-            );
-
-            if (signers.length === 0) {
-                return null;
-            }
-
-            if (signers.length > 1) {
-                throw new Error("Config Error"); // TODO use error as defined by SDK
-            }
+        const signers = await this.fetchCrossmintAPI(
+            "sdk/smart-wallet/signers?type=passkeys",
+            { method: "GET" },
+            "Error fetching passkey validator signer. Please contact support",
+            user.jwt
+        );
 
-            return signers[0].signerData;
-        } catch (e: any) {
-            if (e instanceof CrossmintServiceError && e.status === 404) {
-                return null;
-            }
+        if (signers.length === 0) {
+            return null;
+        }
 
-            throw e;
+        if (signers.length > 1) {
+            throw new SmartWalletSDKError("Passkey Config Error"); // TODO use error as defined by SDK
```

Comment:
can we get a ticket on that `TODO`?

Reply:
Improved the error here instead, don't think we need a specific custom error since it's an internal error.